samples/bot-adaptive-card-actions/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-adaptivecard-actions-csharp",
+    "source": "officeDev",
+    "title": "Get adaptivecard Actions",
+    "shortDescription": "Demonstrating the feature where user can send adaptivecard actions using bot.",
```

Comment:
Demonstrating the feature where user can send adaptive card with actions using bot.

Reply:
fixed